src/plugin/svipc/sysvipc.cpp

```diff
@@ -58,12 +58,10 @@ using namespace dmtcp;
  *  8. Non ckptLeader processes unmap all shmat() addresses corresponding to
  *     the shm-object.
  *  9. BARRIER -- CHECKPOINTED
- * 10. At this point, the contents of the memory-segment have been saved.
```

Comment:
I removed this line because it was related to the barrier, which was not activated. You think I should merge this line with the next one?